CODEOWNERS

```diff
@@ -1,13 +1,2 @@
-# Org-Level Owners (in alphabetical order)
-* @justincormack 
-* @niazfk
-* @stevelasker
-
-# Repo-Level Owners (in alphabetical order)
-# Note: This is only for the notaryproject/notaryproject repo
-* @gokarnm
-* @priteshbandi
-* @shizhMSFT
-* @toddysm
-* @vaninrao10
-* @yizha1
+# Org-Level and repo-Level Owners (in alphabetical order). Please note this is only for the notaryproject/notaryproject repo
+* @gokarnm @justincormack @niazfk @priteshbandi @stevelasker @shizhMSFT @toddysm @vaninrao10 @yizha1
```

Comment:
Please add an empty new line at the end per convention

Reply:
@toddysm Sorry for the late response. What's the reason for adding an empty new line? It seems this syntax rule is not required in https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax